Metallicity_Stack_Commons/valid_table.py

```diff
@@ -46,7 +46,7 @@ def make_validation_table(fitspath: str):
     up_limit = (Hgamma/Hgamma_SN) * 3
 
     valid_stacks_idx = np.where((O_4363_SN >= 3) & (O_5007_SN > 100) & (O_4363_sigma < 1.6))[0] 
-    reliable_5007_stacks = np.where((O_4363_SN < 3) & (O_5007_SN > 100))[0]
+    reliable_5007_stacks = np.where((O_4363_SN < 3) & (O_5007_SN > 120))[0]
```

Comment:
Recall that I want to avoid hard coding for this exact reason that we would need to modify the code. Instead, add an input parameter (e.g., `robust_snr_limit`) that you can specify in `Zcalbase_gal`. The limit that you use can be different from what Caroline will use.